qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -31,54 +31,6 @@
     UPPER_BOUND_DURATION, LOWER_BOUND_DURATION)
 
 
-def get_plot_data_from_segments(segments):
```

Comment:
By deleting this we're changing the API in a backwards-incompatible way, so could potentially break other packages that depend on open controls. I'm not sure how releases of the package are handled -- do we need a version bump or something?

Reply:
Actually this method was never used anywhere and hence should be removed anyway.

Reply:
Not used anywhere by us, but what if there's a third party package depending on it?

Reply:
This was a helper function to another method that actually gets the data for plotting. So the user should use this other method instead. I am pretty sure this should not be a problem.

Reply:
OK, sounds good. As this package matures I expect we'll need to introduce a proper versioning scheme, so that we can have proper deprecation policies for this sort of change, but in these early days I can't imagine too much going wrong.